# Header
std_msgs/Header header

# Data
sensor_msgs/Image color
sensor_msgs/Image depth